imap_processing/codice/codice_l1a.py

```diff
@@ -288,33 +305,23 @@ def get_energy_table(self) -> None:
 
         # Get the appropriate values
         sweep_table = sweep_data[sweep_data["table_idx"] == sweep_table_id]
-        self.energy_table = sweep_table["esa_v"].values
+        energy_table: list[float] = sweep_table["esa_v"].values
```

Comment:
Aren't these numpy arrays? I wonder if we need to bring in typechecking stubs from pandas and/or xarray to help with this... (For another time)
```suggestion
        energy_table: NDArray[float] = sweep_table["esa_v"].values
```

Reply:
Yes, good catch!